2024/json-ld-wg/index.html

```diff
@@ -243,48 +267,54 @@ <h3>
             <dl>
             <dt id="json-ld-12" class="spec">JSON-LD 1.2</dt>
             <dd>
-              <p>This specification will define a new version of JSON-LD, fully compatible with <a href="https://www.w3.org/TR/rdf12-concepts">RDF 1.2</a>.</p>
+              <!-- <p>This specification will define a new version of JSON-LD, fully compatible with <a href="https://www.w3.org/TR/rdf12-concepts">RDF 1.2</a>.</p> -->
+              <p>This specification will define a new version of JSON-LD, including improvements as listed in 
+                the <a href="#scope">Scope Section</a> above. </p>
+                
+                
 
               <p class="draft-status"><b>Draft state:</b> <a href="https://www.w3.org/TR/TR/json-ld11">Adopted from JSON-LD 1.1 Recommendation</a>.</p>
 
               <p class="milestone"><b>Expected completion:</b> Q4 2027.</p>
 
               <p><b>Adopted Draft:</b> JSON-LD 1.1, <a href="https://www.w3.org/TR/2020/REC-json-ld11-20200716/">https://www.w3.org/TR/2020/REC-json-ld11-20200716/</a>, 16 July 2020.</p> 
```

Comment:
```suggestion
              <p><b>Initial Text:</b> JSON-LD 1.1, <a href="https://www.w3.org/TR/2020/REC-json-ld11-20200716/">https://www.w3.org/TR/2020/REC-json-ld11-20200716/</a>, 16 July 2020.</p> 
```

The latest charter template uses "Initial Text" rather than "Adopted Draft" .